client/job.lua

```diff
@@ -409,14 +421,17 @@ else
                 rotation = -20,
                 debug = config.debugPoly,
                 onEnter = function()
+                    if QBX.PlayerData.job.type ~= 'ems' then return end
                     if QBX.PlayerData.job.onduty then
                         lib.showTextUI(locale('text.pstash_button'))
                     end
                 end,
                 onExit = function()
+                    if QBX.PlayerData.job.type ~= 'ems' then return end
```

Comment:
```suggestion
```
Don't let people get the textui stuck if they lose the EMS job while in the zone